### ISSUE TITLE
Error popup when do removeTrack while it loads

### ISSUE THREAD

**Issue by yfedyai** [igvteam/igv.js#1530]
Helo igvteam
igv ver 2.12.2
I am developing a panel to be able show-hide necessary for user tracks, which should be loaded from my own BE API
<img width="318" alt="image" src="https://user-images.githubusercontent.com/33098616/174783791-1324a688-e33d-47d6-8e1c-9f0714b69074.png">
the idea is, that when i click on the checkbox - the track should be loaded, when clicking again - the track should hide 

I'm using this logic for toggling tracks 
```
toggleTrack = (track) => {
            if (track.selected) {
                this.igv_browser.removeTrackByName(track.name)
                track.selected = false;
            } else {
                this.igv_browser.loadTrack(track);
                track.selected = true;
            }
        };
```
everything working ok, as expected, if tracks finished loading and I toggle the checkbox and as a result they appear and disappear 

but I found weird behavior if I uncheck the checkbox(do removeTrackByName) until the track is not fully loaded( folowing by network tab and appeared loading spinner)
I got an popup error and console error( but track line hides)

<img width="405" alt="image" src="https://user-images.githubusercontent.com/33098616/174785110-c1df3e13-8df3-44ae-a913-898b4bc3baa2.png">
<img width="731" alt="image" src="https://user-images.githubusercontent.com/33098616/174785202-087d6949-ba9f-4aca-af2f-9f1ee95216bb.png">

And as I investigate deeper I got such an error when using wig/big, annotation/vcf tracks, NOT for 'alignment' tracks with 'bam/cram' but using the same line of code to show/hide tracks and zero errors 

there is the config for bam tracks

```
let BamTrack = {
              selected: false,
              url: 'myWorkingApi',
              indexURL:'myWorkingApiForIndex,
              name: 'trackNameBam',
              type: 'alignment',
              format: 'bam',
              alignmentRowHeight: 5,
              colorBy: "strand",
              samplingDepth: 500,
              headers: 'MyAPIHeaders',
              removable: false,
            };
```

there is the config for bigwigTracks
```
let bigWigTrack = {
                       selected: false,
                       type: 'wig',
                       format: 'bigwig',
                       name: 'trackName' + ' bigWig',
                       url: 'myWorkingApi',
                       autoscale: true,
                       color: 'rgb(0, 0, 150)',
                       headers: 'MyAPIHeaders',,
                   }
```

any suggestions?
If any additional information needed - please let me know

**Comment by jrobinso** [comment]
This looks like a probable bug in igv.js,  I will look into it.   I don't
think there's anything you can do on your side.

**Comment by jrobinso** [comment]
Apologies for the delay, I was interrupted by travel.   This is partially fixed but there are still some small issues, I hope to have a fix out next week.

**Comment by yfedyai** [comment]
@jrobinso ok, no problem, take your time

**Comment by jrobinso** [comment]
Fixed with commit 33b5fbf7
